spec/controllers/forgot_passwords_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe ForgotPasswordsController, :type => :controller do 
+
+  describe "POST create" do
+    context "with blank input" do
+      it "redirects to the forgot password page" do
+        post :create, email: ''
+        expect(response).to redirect_to forgot_password_path
+      end
+
+      it "shows an error message" do
+        post :create, email: ''
+        expect(flash[:error]).to eq("Email cannot be blank.")
```

Comment:
I would not recommend testing exact flash error here. Feature specs are better place for this.